nntrainer/include/weight.h

```diff
@@ -168,6 +181,7 @@ class Weight {
    */
   Tensor &getGradientRef() { return grad; }
 
+private:
```

Comment:
unnecessary change

Reply:
This should have gone to a separate patch. I'll move this after the discussion. 

I am thinking to open this up for the test. Is there a reason Ref 's are private?

Reply:
Yes, so that any other class other the friend class cannot directly update this as an operation. 
I know `float * data` is accessible, but if someone is changing that, its intentional.
Getting weight by reference is only allowed for object owners which is this case is the layer and its optimizer.

Reply:
The change has been reverted. I'll use data for now.